.github/workflows/publish.template.yaml

```diff
@@ -14,7 +14,7 @@ on:
       pr-number:
         type: number
         description: When the event is a pull-request it passes the pr number
-        required: true
+        required: false
```

Comment:
Do you want to check the existence of this input where you use it now? Otherwise you will get unexpected consequences